cmd/minikube/cmd/start.go

```diff
@@ -102,6 +103,15 @@ func runStart(cmd *cobra.Command, args []string) {
 	kubeHost = strings.Replace(kubeHost, ":2376", ":"+strconv.Itoa(constants.APIServerPort), -1)
 	fmt.Printf("Kubernetes is available at %s.\n", kubeHost)
 
+	//Checking for kubectl
+	fmt.Printf("Checking for Kubectl ...\n")
+	if _, err := exec.LookPath("kubectl") {
+  		glog.Errorln("For more info on kubectl visit http://kubernetes.io/docs/getting-started-guides/minikube/#download-kubectl")
+  		os.Exit(1)
```

Comment:
Could you drop the os.Exit(1) here? I think this should just log a warning instead of an error.